airflow/api/common/airflow_health.py

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from airflow.jobs.scheduler_job_runner import SchedulerJobRunner
+from airflow.jobs.triggerer_job_runner import TriggererJobRunner
+
+HEALTHY = "healthy"
+UNHEALTHY = "unhealthy"
+
+
+def get_airflow_health():
```

Comment:
Maybe we could add a return type annotation. (TypedDict ?)

Reply:
I tried to be descriptive by specifying the return type as `dict[str, dict[str, str | None]]`, but it failed both locally (I thought my breeze might be acting weird) and also on CI with the below error, so changed it to `dict[str, Any]` which breeze accepted. So, hopefully CI will accept it too 🤞🏽 .

<img width="1269" alt="Screenshot 2023-05-27 at 12 04 31 AM" src="https://github.com/apache/airflow/assets/10206082/46222cb8-0af8-46a5-beb7-67308f2cde79">

Reply:
yes, CI is green now.

Reply:
I think you missed a nested level of dict, but I think any is acceptable here too.